src/test/polymer/polymer-element-scanner_test.ts

```diff
@@ -73,7 +75,10 @@ suite('PolymerElementScanner', () => {
         observers: [
           '_anObserver()',
           '_anotherObserver()'
-        ]
+        ],
+        listeners: {
+          'event-a': '_handleA'
```

Comment:
Add 1 more event to make sure it handles multiple? Also maybe discard bogus events